apps/prairielearn/src/sync/fromDisk/courseInstances.js

```diff
@@ -31,7 +30,7 @@ function getParamsForCourseInstance(courseInstance, courseTimezone) {
     long_name: courseInstance.longName,
     number: courseInstance.number,
     hide_in_enroll_page: courseInstance.hideInEnrollPage || false,
-    display_timezone: courseInstance.timezone || courseTimezone || 'America/Chicago',
+    display_timezone: courseInstance.timezone || null,
```

Comment:
Nit, I generally reach for the nullish-coalescing operator for things like this:

```suggestion
    display_timezone: courseInstance.timezone ?? null,
```

Reply:
The `||` would take care of blank timezones, though...